lessons/201/java-app-reactive/pom.xml

```diff
@@ -0,0 +1,152 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 https://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+    <groupId>com.antonputra</groupId>
+    <artifactId>java-app</artifactId>
+    <version>0.1.0</version>
+
+    <properties>
+        <compiler-plugin.version>3.13.0</compiler-plugin.version>
```

Comment:
Consider a 'quarkus update' . I want to say I remember reading about some improvements to jackson in 3.14+

Reply:
thanks for the tip, I'll update quarkus version before next test

Reply:
to get the hackson improvements from quarkus 3.14 there is also the need to seed the property below

```
quarkus.rest.jackson.optimization.enable-reflection-free-serializers=true
```